#Targets list for the shadow etherCAT hand
targets :
       fist_step1 : 
                   THJ1 : 68 
                   THJ2 : 30 
                   THJ3 : 15 
                   THJ4 : 70
                   THJ5 : 30
                   FFJ0 : 0
                   FFJ3 : 0
                   FFJ4 : 0
                   MFJ0 : 0 
                   MFJ3 : 0
                   MFJ4 : 0  
                   RFJ0 : 0
                   RFJ3 : 0
                   RFJ4 : 0
                   LFJ0 : 0
                   LFJ3 : 0
                   LFJ4 : 0
                   LFJ5 : 0
                   WRJ1 : 0  
                   WRJ2 : 0
       fist_step2 : 
                   THJ1 : 68 
                   THJ2 : 30 
                   THJ3 : 15 
                   THJ4 : 70
                   THJ5 : 30
                   FFJ0 : 118
                   FFJ3 : 45
                   FFJ4 : 0
                   MFJ0 : 118
                   MFJ3 : 45
                   MFJ4 : 0  
                   RFJ0 : 118
                   RFJ3 : 45
                   RFJ4 : 0
                   LFJ0 : 118
                   LFJ3 : 45
                   LFJ4 : 0
                   LFJ5 : 0
                   WRJ1 : 0  
                   WRJ2 : 0
       
       hand_extended_pos : &id001
                           THJ1 : 21 
                           THJ2 : 25 
                           THJ3 : 0 
                           THJ4 : 15
                           THJ5 : 9
                           FFJ0 : 0
                           FFJ3 : 0
                           FFJ4 : 0
                           MFJ0 : 0
                           MFJ3 : 0
                           MFJ4 : 0  
                           RFJ0 : 0
                           RFJ3 : 0
                           RFJ4 : 0
                           LFJ0 : 0
                           LFJ3 : 0
                           LFJ4 : 0
                           LFJ5 : 0
                           WRJ1 : 0  
                           WRJ2 : 0
       one : 
                   THJ1 : 68 
                   THJ2 : 30 
                   THJ3 : 15 
                   THJ4 : 70
                   THJ5 : 30
                   FFJ0 : 0
                   FFJ3 : 0
                   FFJ4 : 0
                   MFJ0 : 118
                   MFJ3 : 45
                   MFJ4 : 0  
                   RFJ0 : 118
                   RFJ3 : 45
                   RFJ4 : 0
                   LFJ0 : 118
                   LFJ3 : 45
                   LFJ4 : 0
                   LFJ5 : 0
                   WRJ1 : 0  
                   WRJ2 : 0
       two : 
                   THJ1 : 68 
                   THJ2 : 30 
                   THJ3 : 15 
                   THJ4 : 70
                   THJ5 : 30
                   FFJ0 : 0
                   FFJ3 : 0
                   FFJ4 : 0
                   MFJ0 : 0
                   MFJ3 : 0
                   MFJ4 : 0  
                   RFJ0 : 118
                   RFJ3 : 45
                   RFJ4 : 0
                   LFJ0 : 118
                   LFJ3 : 45
                   LFJ4 : 0
                   LFJ5 : 0
                   WRJ1 : 0  
                   WRJ2 : 0
       three : 
                   THJ1 : 68 
                   THJ2 : 30 
                   THJ3 : 15 
                   THJ4 : 70
                   THJ5 : 30
                   FFJ0 : 0
                   FFJ3 : 0
                   FFJ4 : 0
                   MFJ0 : 0
                   MFJ3 : 0
                   MFJ4 : 0  
                   RFJ0 : 0
                   RFJ3 : 0
                   RFJ4 : 0
                   LFJ0 : 118
                   LFJ3 : 45
                   LFJ4 : 0
                   LFJ5 : 0
                   WRJ1 : 0  
                   WRJ2 : 0
       four : 
                   THJ1 : 68 
                   THJ2 : 30 
                   THJ3 : 15 
                   THJ4 : 70
                   THJ5 : 30
                   FFJ0 : 0
                   FFJ3 : 0
                   FFJ4 : 0
                   MFJ0 : 0
                   MFJ3 : 0
                   MFJ4 : 0  
                   RFJ0 : 0
                   RFJ3 : 0
                   RFJ4 : 0
                   LFJ0 : 0
                   LFJ3 : 0
                   LFJ4 : 0
                   LFJ5 : 0
                   WRJ1 : 0  
                   WRJ2 : 0
       five : *id001
                   
